src/components/Layout/ScrollableParent.tsx

```diff
@@ -29,8 +32,9 @@ export function ScrollableParent({ children, xss }: PropsWithChildren<Scrollable
        * Otherwise, the flex-item's min-height/width is based on the content of the flex-item, which maybe overflow the container.
        * See https://stackoverflow.com/questions/42130384/why-should-i-specify-height-0-even-if-i-specified-flex-basis-0-in-css3-flexbox */}
       <div css={{ ...Css.mh0.mw0.df.fdc.$, ...xss }}>
-        {children}
-        <div css={Css.overflowAuto.$} ref={scrollableRef}></div>
+        <div css={Css.pxPx(context.paddingX).$}>{children}</div>
```

Comment:
Ah, I see, I was wondering about, instead of a new `paddingX` prop, we just tried to pulling the padding from the `xss`, but you're really setting the padding in here, and not on the root element...

Still, kind of wonder if we could do that...

Like the caller does `ScrollableParent xss=Css.px1.$` which gets passed in as having `padding-left` and `padding-right` set, but we a) see that they're set and put them into the context value, b) unset the in the `xss` and c) actually set it on this div right here...

I dunno, probably too fancy, but would be cute...

Reply:
Yea, I get what you're saying. I wonder if that'd would be a bit much. Because we'd need to check for the existence of both paddingLeft and paddingRight, and then check they are the same value to assume this is what the user really wants to do.

Reply:
Granted, I guess we don't need to see that they are the same,.. we can use each individually and have a `pl` value and a `pr` value.